### ISSUE TITLE
[e2e failure] [sig-cli] Kubectl client [k8s.io] Kubectl describe should check if kubectl describe prints relevant information for rc and pods [Conformance]

### ISSUE THREAD

**Issue by spiffxp** [kubernetes/kubernetes#56414]
/priority critical-urgent
/priority failing-test
/kind bug
/status approved-for-milestone
@kubernetes/sig-cli-test-failures 

This test has been failing since at least 2017-11-23 in the following jobs:
- https://k8s-testgrid.appspot.com/sig-release-master-upgrade#gce-1.8-master-upgrade-master
- https://k8s-testgrid.appspot.com/sig-release-master-upgrade#gce-1.8-master-upgrade-cluster-parallel
- https://k8s-testgrid.appspot.com/sig-release-master-upgrade#gce-master-1.8-downgrade-cluster-parallel
- https://k8s-testgrid.appspot.com/sig-release-master-upgrade#gke-gci-master-gci-1.8-downgrade-cluster-parallel

These jobs are on the [sig-release-master-upgrade dashboard](https://k8s-testgrid.appspot.com/sig-release-master-upgrade), and prevent us from cutting [v1.9.0-beta.1] (kubernetes/sig-release#34). Is there work ongoing to bring this test back to green?

triage cluster [9171dbbaaa771d52dba5](https://storage.googleapis.com/k8s-gubernator/triage/index.html?test=%5C%5Bsig%5C-cli%5C%5D%5C%20Kubectl%5C%20client%5C%20%5C%5Bk8s%5C.io%5C%5D%5C%20Kubectl%5C%20describe%5C%20should%5C%20check%5C%20if%5C%20kubectl%5C%20describe%5C%20prints%5C%20relevant%5C%20information%5C%20for%5C%20rc%5C%20and%5C%20pods%5C%20%5C%5BConformance%5C%5D#9171dbbaaa771d52dba5)
```
/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/kubectl/kubectl.go:934
Nov 21 15:24:34.066: failed to find Created By: in Name:           redis-master-fjtpl
Namespace:      e2e-tests-kubectl-h9s9j
```

- also failing in:
  - https://k8s-testgrid.appspot.com/sig-release-master-kubectl-skew#skew-clustermaster-kubectl1.8-gce
  - https://k8s-testgrid.appspot.com/sig-release-master-kubectl-skew#skew-cluster1.8-kubectl1.8-gke
- Related issues or PRs:
  - https://github.com/kubernetes/kubernetes/issues/43464
  - https://github.com/kubernetes/kubernetes/issues/41898

**Comment by mengqiy** [comment]
@stewart-yu Thanks for looking into this issue. Let me know if you need help. My slack name is the same as my GitHub handle.

**Comment by mengqiy** [comment]
The test is failing due to the change in https://github.com/kubernetes/kubernetes/pull/42849

**Comment by mengqiy** [comment]
Sent a fix at #56508.

**Comment by janetkuo** [comment]
> The test is failing due to the change in #42849

@mengqiy for the record, this is not caused by #42849, which is merged in 1.7. 

The is affected by #54445 (1.9), where created-by annotation generation is removed from server side: https://github.com/kubernetes/kubernetes/pull/54445/files#diff-44bba2c68a6ea2b4ca0dbb7f4465828eL498

**Comment by jberkus** [comment]
If this is fixed, can we close the issue?
